content/docs/atom-guide/look-dev/materials/materials.md

```diff
@@ -0,0 +1,79 @@
+
+For more information, see [Material File Specification](/docs/atom-guide/look-dev/materials/material-file-spec/).  
+
+### Material File Example
+
+In the following example, the material uses the **StandardPBR** material type and contains the properties `baseColor`, `roughness`, and `normal`. 
+
+```json
+{
+    "materialType": "Materials/Types/StandardPBR.materialtype",
+    "materialTypeVersion": 6,
+    "propertyValues": {
+        "baseColor.textureMap": "Textures/Default/default_basecolor.tif",
+        "roughness.textureMap": "Textures/Default/default_roughness.tif",
+        "normal.factor": 0.8,
+        "normal.textureMap": "Textures/Default/default_normal.tif"
+    }
+}
+```
+
+{{< image-width "/images/atom-guide/materials/simple-example.jpg" "720" "Example Material Screenshot" >}}
+
+{{< note >}}
+The textures in the sample above can be found in `Gems/Atom/Feature/Common/Assets/Textures/Default`.
+{{< /note >}}
+
+## Material Types
+A **material type** is a data item that includes everything that's needed to describe how to render a mesh:  
+- A set of *material property* definitions
+- Links to shaders
+- Scripts that describe how to consume the material properties
+- Metadata for the Material Editor to use when displaying material properties
+
+Material type files (`*.materialtype`) are in JSON format and are authored directly. This will link together several other files as well, to form a complete material type definition. These additional files include *.shader*, *.azsl*, and *.lua* files.
+
+{{< note >}}
+A node graph based tool for authoring material types is in development, but until that's available material types are authored directly.
+{{< /note >}}
+
+For more information, see [Material Type Authoring Guide](/docs/atom-guide/dev-guide/materials/material-type-authoring-guide/).  
+
+## Atom PBR Material Types
+*Physically based rendering (PBR)* is a rendering technique that simulates realistic interaction between materials and light. Atom provides several core material types based on industry standard workflows using metallic and roughness properties. Atom's material system is also capable of rendering non-photo-realistic techniques and special effects through alternate material types; users can author their own material types to suit their needs.
+
+For more information on the available PBR material types, see [Physically-based Rendering (PBR)](/docs/atom-guide/look-dev/materials/pbr/)<!-- and [Working with StandardPBR materials](./material-build-pipeline)DRAFT TOPIC-->. 
+
+## Material Assets and Material Type Assets
+Material and material type files produce **Material Assets** and **Material Type Assets** in the cache, respectively. The runtime loads these asset files and uses them to render models. 
```

Comment:
I think some general context should be added here to orient users. To start, introducing the Asset Processor and it's role here. Plus, your diagram would already be a great supplement to that information as well. 

Questions from a user perspective might be: 
- Who's responsible for producing the assets? Does this happen automatically or do I need to do something?
- When does this get triggered? (e.g. when the user opens the Editor, or another runtime)	
  - Support for hot-reloading?
		
If it seems fitting, you can also consider renaming this section to something like "How materials are processed". ("Processing materials" is more straightforward, but starter *-ing* verb often indicate user action, whereas here you're explaining a concept to the user. )

Reply:
Okay, I'll add a note with a link to a page about asset processing.